kivy/core/window/_window_sdl2.pyx

```diff
@@ -686,3 +688,14 @@ cdef SDL_Surface* flipVert(SDL_Surface* sfc):
         memcpy(&rpixels[pos], &pixels[(pxlength - pos) - pitch], pitch)
 
     return result
+
+cdef SDL_Surface* surface_from_core_image(core_image):
+    # create an SDL_Surface from a core image's texture pixels. Use
+    # SDL_FreeSurface(surface) to deallocate
+    width, height = core_image.size
```

Comment:
I think these could be `cdef int`s since it's already a `cdef`ed function anyway.